snuba/datasets/cdc/groupedmessage_processor.py

```diff
@@ -17,12 +22,27 @@ class GroupMessageRecord:
     first_release_id: Optional[int] = None
 
 
+@dataclass(frozen=True)
+class RawGroupMessageRecord:
+    """
+    This is a faster verison of GroupMessageRecord that does
+    not rely on datetime objects. This is useful for bulk load
+    of massive tables to avoid creating and serializing datetime
+    objects at every record.
```

Comment:
Is there a drawback to just using this approach consistently?

Reply:
For groupedmessage it does not matter. Other the processor in other datasets can do math based on time so in that case it is better to have a datetime object. Overall I wanted to test the improvement in using this for everything separately (it is a different test to do)